models/members.js

```diff
@@ -17,7 +17,23 @@ var app = app || {};
 
   Member.fetchMember = (lName) => $.getJSON(_API_URL_ + '/api/v0/members/' + lName).then(data => console.log(data)).catch(err => console.error(errorCallback(err)));
 
-  Member.addMember = (member) => $.post(_API_URL_ + '/api/v0/members/add', member).catch(err => console.error(errorCallback(err)));
+  Member.addMember = (member) => $.ajax(
```

Comment:
I see that addMember is a function defined as a private property of member module.  But there is no return path to the caller.  It's getting the response and logging to console.

Reply:
How would you write the return path to the caller? So far I've only been trying to make sure I can post to the database and then I might add in a callback that will tell the user their post.